java-sdk-example/src/test/java/CreateSandboxTestWithCustomPatch.java

```diff
@@ -0,0 +1,136 @@
+import com.signadot.ApiClient;
+import com.signadot.ApiException;
+import com.signadot.api.SandboxesApi;
+import com.signadot.model.*;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+
+public class CreateSandboxTestWithCustomPatch {
+
+  public static final String HOTROD = "hotrod";
+  public static final String CUSTOM_PATCH = "spec:\n" +
+                                            "  template:\n" +
+                                            "    spec:\n" +
+                                            "      containers:\n" +
+                                            "      - name: hotrod\n" +
+                                            "        env:\n" +
+                                            "        - name:  PATCH_TEST_VAR\n" +
+                                            "          value: foo\n";
```

Comment:
Not totally necessary, but it'd be nicer to set a random value every time and test against that. That way we can be confident that the value in the response is actually being set from env var, and not just hard coded to "foo".